packages/experimental/src/experimental-list/style.scss

```diff
@@ -0,0 +1,158 @@
+.woocommerce-experimental-list {
```

Comment:
This is copy of the `style.scss` within `components/src/list`, but we needed it also here as the `experimental` package can be installed as a stand alone.
To avoid overlapping css I changed the classname from `woocommerce-list` to `woocommerce-experimental-list`.